public/js/accordion.js

```diff
@@ -1,15 +1,15 @@
-window.schuldhulp = window.schuldhulp || {};
-
-document.addEventListener('click', function (event) {
-    if (window.schuldhulp._.matches(event.target, '.accordion .accordion-header a, .accordion .accordion-header a *')) {
-        event.preventDefault();
-        var accordionDom = window.schuldhulp._.findParent(event.target, '.accordion');
-        var accordionContainer = window.schuldhulp._.findParent(event.target, '.accordion-container');
-        var isOpen = window.schuldhulp._.matches(accordionDom, '.open');
-        if (isOpen === true) {
-            accordionDom.classList.remove('open');
-        } else {
-            accordionDom.classList.add('open');
-        }
-    }
-});
+window.schuldhulp = window.schuldhulp || {};
+
+document.addEventListener('click', function (event) {
+    if (window.schuldhulp._.matches(event.target, '.accordion .accordion-header a, .accordion .accordion-header a *')) {
+        event.preventDefault();
+        var accordionDom = window.schuldhulp._.findParent(event.target, '.accordion');
+        var accordionContainer = window.schuldhulp._.findParent(event.target, '.accordion-container');
+        var isOpen = window.schuldhulp._.matches(accordionDom, '.open');
+        if (isOpen === true) {
+            accordionDom.classList.remove('open');
+        } else {
+            accordionDom.classList.add('open');
+        }
+    }
+});
```

Comment:
Hier zijn de codewijzigingen niet leesbaar. Als er geen codewijzigingen zijn, moet dit stuk er uit of anders alleen de diffs tonen.

Reply:
Zie mijn [algemene reactie](https://github.com/Amsterdam/schulddossier/pull/277#issuecomment-1097824167), er is gelukkig een mogelijkheid in GitHub aanwezig om deze whitespace change weg te filteren.